2-mandatory/4-people-i-know.js

```diff
@@ -394,6 +408,20 @@ This time, I only want the full names ("<firstname> <lastname>") of my friends w
 */
 
 let friendsWhoAreColleaguesOfStacie = [];
+for (let key in friends) {
+  if (
+    friends[key].colleagues.find(
+      (colleageNames) => colleageNames.name === "Stacie Villarreal"
```

Comment:
I probaly would have assigned the result of find() to a var for readability (probaly would have used contains() or indexOf though)